packages/context/src/inject-config.ts

```diff
@@ -0,0 +1,167 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/context
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {BindingFilter} from './binding-filter';
+import {BindingKey} from './binding-key';
+import {Context} from './context';
+import {ContextView} from './context-view';
+import {
+  inject,
+  Injection,
+  InjectionMetadata,
+  inspectTargetType,
+} from './inject';
+import {ResolutionSession} from './resolution-session';
+import {getDeepProperty, ValueOrPromise} from './value-promise';
+/**
```

Comment:
Please add an empty line to visually delimit the block of `import` statements from the actual code.